### PR TITLE
 creating actions for meetings 

### PR DESCRIPTION
**Pull request by JorPoon**
# Description

Creating actions and action types for meetings, axios endpoints are not definite and will meed to match backend.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status

- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?

Has not been tested yet, so far there are no errors in the console. 

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by tfolbrecht — DISMISSED** [review]

File: src/actions/meetingActions.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+
+//backend  meetings url
+const url = "http://mentorbe.tfolbrecht.com/";
+
+export const GET_MEETINGS_START = "GET_MEETINGS_START";
+export const GET_MEETINGS_SUCCESS = "GET_MEETINGS_SUCCESS";
+export const GET_MEETINGS_FAILURE = "GET_MEETINGS_FAILURE";
```
Comment (Estrax) [inline src/actions/meetingActions.js]: let's move action types to /src/constants/actionTypes.js

File: src/actions/meetingActions.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+export const DELETE_MEETING_SUCCESS = "DELETE_MEETING_SUCCESS";
+export const DELETE_MEETING_FAILURE = "DELETE_MEETING_FAILURE";
+
+export const deleteMeeting = id => dispatch => {
+    dispatch({type: DELETE_MEETING_START});
+    return axios
+        .delete(`${url}/meetings/${id}`)
+        .then(res => {
+            console.log(res);
+            dispatch({type: DELETE_MEETING_START, payload:id})
```
Comment (Estrax) [inline src/actions/meetingActions.js]: shouldn't it be `DELETE_MEETING_SUCCESS` instead?

File: src/actions/meetingActions.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+export const DELETE_MEETING_SUCCESS = "DELETE_MEETING_SUCCESS";
+export const DELETE_MEETING_FAILURE = "DELETE_MEETING_FAILURE";
+
+export const deleteMeeting = id => dispatch => {
+  dispatch({ type: DELETE_MEETING_START });
+  return axios
+    .delete(`${url}/meetings/${id}`)
+    .then(res => {
+      console.log(res);
+      dispatch({ type: DELETE_MEETING_START, payload: id });
```
Comment (Estrax) [inline src/actions/meetingActions.js]: shouldn't it be `DELETE_MEETING_SUCCESS` instead?

**Review by Estrax — APPROVED** [review]

**Review by Estrax — DISMISSED** [review]

**Review by Estrax — DISMISSED** [review]

**Review by jdspell — CHANGES_REQUESTED** [review]
Code looks good. Just separate out the action constants and the action creators.

**Review by jdspell — DISMISSED** [review]

**Review by tfolbrecht — APPROVED** [review]
an env variable for backend url would be better, will add on my my PR!

**Review by Estrax — APPROVED** [review]

**Review by jdspell — APPROVED** [review]

**Review by jdspell — COMMENTED** [review]
line 47- need to include meeting id in params

line 64- do not need backslash before meetings as the url above already ends in a backslash

**Pull request merged**